Makefile

```diff
@@ -143,15 +143,15 @@ clean:
 
 .PHONY: codegen
 codegen:
-	go build -o ${KOPS_ROOT}/_output/bin k8s.io/kops/upup/tools/generators/...
+	go build -o ${KOPS_ROOT}/_output/bin/ k8s.io/kops/upup/tools/generators/...
```

Comment:
Without this change to the `codegen` and `verify-codegen` targets, on my system, I get the `fitask` binary at `${KOPS_ROOT}/_output/bin` instead of `${KOPS_ROOT}/_output/bin/fitask`.